src/components/navBar/index.tsx

```diff
@@ -19,9 +23,10 @@ import { useGetUserQuery } from '@/app/services/userApi';
 import { Loader } from '../tasks/card/Loader';
 
 const NavBar = () => {
+    const dispatch = useDispatch();
     const isLoggedIn = true;
     const { data: userData } = useGetUserQuery();
-    // const { userData, isLoggedIn } = useAuthenticated();
+    dispatch(updateUser(userData));
```

Comment:
Arsalan please read on RTK Query you don't have to add user reducers, your data stays stored in the query

Reply:
Okay I will look into it , thanks!